modules/gapi/misc/python/test/test_gapi_core.py

```diff
@@ -37,6 +37,23 @@ def test_add(self):
             # Comparison
             self.assertEqual(0.0, cv.norm(expected, actual, cv.NORM_INF))
 
+    def test_mean(self):
+        sz = (1280, 720, 3)
+        in1 = np.random.randint(0, 100, sz).astype(np.uint8)
+
+        # OpenCV
+        expected = np.mean(in1,axis=(0,1))
```

Comment:
BTW, it isn't `opencv` reference it is `numpy`, does it make sense to compare with opencv ?

Reply:
G-API is positioned as optimized way to rewrite OpenCV algorithms so I think it's useful to see alternatives from OpenCV core, right.

Reply:
Fixed